lms/static/sass/_pearsonx.scss

```diff
@@ -0,0 +1,132 @@
+@import 'lms/static/sass/multicourse/home';
```

Comment:
Ah, no, it should not be. It took me a few tries to figure out the right things to override and the right things to import.